modules/rosapps/applications/screensavers/mazescr/lang/ro-RO.rc

```diff
@@ -1,47 +1,53 @@
-/* Translator: Ștefan Fulea (stefan dot fulea at mail dot com) */
+ /*
+ * PROJECT:     Maze ScreenSaver
+ * LICENSE:     HPND (https://spdx.org/licenses/HPND)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
 
 IDD_DLG_SCREEN DIALOGEX DISCARDABLE  0, 0, 276, 112
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Configurație animație de inactivitate „Labirint”"
+CAPTION "Setări pentru economizorul de ecran Labirint"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT           "Mărime labirint:",IDC_STATIC,7,7,90,12,SS_CENTERIMAGE
+    LTEXT           "Mărimea Labirintului:",IDC_STATIC,7,7,90,12,SS_CENTERIMAGE
     CONTROL         "Slider1",IDC_SLIDER_SIZE,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,101,7,140,12
     CTEXT           "Static",IDC_TEXT_SIZE,245,7,24,12,SS_CENTERIMAGE,WS_EX_STATICEDGE
-    LTEXT           "Timp ante-rezolvare:",IDC_STATIC,7,28,90,12,SS_CENTERIMAGE
+    LTEXT           "Întârziere de pre-rezolvare:",IDC_STATIC,7,28,90,12,SS_CENTERIMAGE
     CONTROL         "Slider2",IDC_SLIDER_PRESD,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,101,28,140,12
     CTEXT           "Static",IDC_TEXT_PRESD,245,28,24,12,SS_CENTERIMAGE,WS_EX_STATICEDGE
-    LTEXT           "Timp post-rezolvare:",IDC_STATIC,7,49,90,12,SS_CENTERIMAGE
+    LTEXT           "Întârziere de post-rezolvare:",IDC_STATIC,7,49,90,12,SS_CENTERIMAGE
```

Comment:
Prove with a pic that your enlarged string does still fit the 90 width!

Reply:
I cannot find Maze Star Screensaver. Can you help me?

Reply:
the file gets only built when you `configure -DENABLE_ROSAPPS=1` like that in your workspace. Something that you always should do when touching rosapps-components. Typically you should always build yourself before opening a PR.

Reply:
In spite of showing me, I still don't know how to do it. Meanwhile I abbreviated the word "Întârziere".

Reply:
I can live with that solution. (Still I wonder why you didn't just stick with the old translation). Nvm. Approved now with the shortened strings.

Reply:
I did this change because "delay"delay means "întârziere"întârziere and not "timp"timp. This translation is wrong. The word "time" means "timp" in Romanian.